src/Strategy/Python/ReqTxt.hs

```diff
@@ -34,22 +34,18 @@ reqParser = [] <$ char '-' <* ignored -- pip options
      <|> [] <$ char '/' <* ignored -- absolute path
      <|> [] <$ oneOfS ["http:", "https:", "git+", "hg+", "svn+", "bzr+"] <* ignored -- URLs
      <|> [] <$ comment
-     <|> (pure <$> requirementParser <* ignored)
-     <|> pure [] -- empty line
+     <|> (pure <$> try requirementParser <* ignored)
+     <|> [] <$ ignored -- something else we're not able to parse
 
   where
   isEndLine :: Char -> Bool
   isEndLine '\n' = True
   isEndLine '\r' = True
-  isEndLine '\\' = True
   isEndLine _    = False
 
   -- ignore content until the end of the line
   ignored :: Parser ()
-  ignored = () <$ takeWhileP (Just "ignored") (not . isEndLine) <* (try newLine <|> (() <$ takeWhileP (Just "end of line") isEndLine))
-
-  newLine :: Parser ()
```

Comment:
I'm curious if removing this newLine parsing will ever come back to bite us, although I don't see any difference between the way I implemented it and the way you are handling it. If we ever want to capture the data in the newlines, such as `hash` like in the test file, this may come in handy, but until then I don't care.